uswds/uswds
# [Beta] Sync 2.13.1

## Preview
[Federalist preview →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/jm-beta-sync-2.13.1/)

## Description

**Beta is now on 2.13.1**. Latest changes from release [2.13.1 ](https://github.com/uswds/uswds/releases/tag/v2.13.1) have been included in the beta. Closes #4544.

## Additional information

No additional updates are required from the user.

- [x] Sync to 2.13.0
- [x] Sync to 2.13.1

## How to test 🧪
- NPM scripts should run without errors
- Verify there are no lint errors
- StorybookJS runs and components look as expected

Before you hit Submit, make sure you’ve done whichever of these applies to you:
